tasks.py

```diff
@@ -125,7 +125,7 @@ def lint(
     unsafe_fixes: bool = False,
     output_format: Literal["full", "concise", "github"] | None = None,
     watch: bool = False,
-    pty: bool = True,
+    pty: bool = False,
```

Comment:
this should be reverted - it allows colorized output of the ruff command to the terminal. If for some reason this is causing problems for you locally, you could try ` invoke lint --no-pty`.